### PR TITLE
Fixed GetAllSports endpoint

### PR DESCRIPTION
**Pull request by melanieet**

### CODE REVIEW

File: all_files/backend_not_separated.py [hunk 1]
```diff
@@ -63,7 +63,7 @@ def present_schedule(list_of_formatted_entries):
```
Comment (ellietms) [inline all_files/backend_not_separated.py:64]: Hi @melanieet  , Thanks for the update. I am wondering what is the reason you changed the whole camel case variable names and changed them to _ ?

Comment (melanieet) [reply all_files/backend_not_separated.py:64]: Hi @ellietms,
No worries! I think it's [Python convention](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names) not to use camelcase for variable and function names.
Edited to add: It doesn't look quite so good, does it?

Comment (ellietms) [reply all_files/backend_not_separated.py:64]: @melanieet  no looks good, but I just saw it's megs codes. I actually  updated the main branch , could you please merge the changes and update the variables (if you would like to change them)

**Review by ellietms — CHANGES_REQUESTED** [review]

**Review by megdye — APPROVED** [review]
Agree with all of it!

File: all_files/backend_not_separated.py [hunk 2]
```diff
@@ -3,9 +3,9 @@
 from datetime import datetime, timedelta
```
Comment (ellietms) [inline all_files/backend_not_separated.py:4]: @melanieet  please feel free to close your pull request 🌹

**Review by ellietms — CHANGES_REQUESTED** [review]

**Pull request closed without merging**